test/lib/ufe/testShaderNodeDef.py

```diff
@@ -93,8 +94,8 @@ def testDefinitions(self):
         for mtlxDef in mtlxDefs:
             self.assertTrue(mtlxDef in nodeDefTypes)
 
-    @unittest.skipIf(os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') < '4001', 'nodeDefHandler is only available in UFE preview version 0.4.1 and greater')
-    @unittest.skipIf(os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') >= '4015', 'filename and float2 are no longer generic starting with Ufe 0.4.15')
+    @unittest.skipUnless(ufeUtils.ufeFeatureSetVersion() >= 4, 'nodeDefHandler is only available in UFE v4 or greater')
+    @unittest.skipIf(ufeUtils.ufeFeatureSetVersion() >= 4, 'filename and float2 are no longer generic starting with Ufe v4')
```

Comment:
Something seems off here. I think we might just need the test below (line 143). See original PR where this was split https://github.com/Autodesk/maya-usd/commit/4f34c98259da09542763822103f5f525fa985c0d#diff-704929d4dbdc0ef77e39bdc75501acb1e470f82877cad0f5eaab20a27823e35b

Reply:
Yes my bad. The testDefinitionByType test is only for 4001 to 4015. So we only need to run testDefinitionByType.